### PR TITLE
Feat: Advanced Options 

### PR DESCRIPTION
**Pull request by red-game-dev**
Ticket: https://github.com/orgs/HyperPlay-Gaming/projects/1/views/1?pane=issue&itemId=54670406

Preview:
image

image

---

Use the following Checklist if you have changed something on the Backend or Frontend:

- [ ] Tested the feature and it's working on a current and clean install.
- [ ] Tested the main App features and they are still working on a current and clean install. (Login, Install, Play, Uninstall, Move games, etc.)
- [ ] Created / Updated Tests (If necessary)
- [ ] Created / Updated documentation (If necessary)

### CODE REVIEW

**Comment by red-game-dev** [comment]
At some point on this, I was thinking of moving it to the `@hyperplay/ui` component package, but this could be enhanced later because we wouldn't be required to edit this repository other than upgrade the package for these instances.

**Comment by BrettCleary** [comment]
@red-game-dev I think we should change the text to "Show Advanced Settings" if collapsed and "Hide Advanced Settings" if expanded

**Comment by red-game-dev** [comment]
> @red-game-dev I think we should change the text to "Show Advanced Settings" if collapsed and "Hide Advanced Settings" if expanded

Yup make sense, on it 🙏

File: src/frontend/screens/Onboarding/walletSelection/screens/import/components/ImportAndCreateOptions/index.tsx [hunk 1]
```diff
@@ -27,30 +33,44 @@ export default function ImportAndCreateOptions({
...
-      {importableBrowserOptions}
-      <ImportOption
-        override="create"
-        title={t(
-          'hyperplay.onboarding.walletSelection.screens.import.createNewWallet',
-          `Create new MM extension wallet`
+    <>
+      <div className={styles.importOptionsContainer}>
+        {importableBrowserOptions}
+        {isOpenAdvancedOptions && (
```
Comment (BrettCleary) [inline src/frontend/screens/Onboarding/walletSelection/screens/import/components/ImportAndCreateOptions/index.tsx:39]: can we make this a ternary https://medium.com/geekculture/stop-using-for-conditional-rendering-in-react-a0f7b96200f8

**Review by BrettCleary — CHANGES_REQUESTED** [review]
Requesting changes for my comment above for the button text

**Comment by BrettCleary** [comment]
were we waiting on designs for this one @red-game-dev ?

**Comment by red-game-dev** [comment]
> were we waiting on designs for this one @red-game-dev ?

Ye, but  @iamdavid0091 replied with the changes needs to be applied, https://github.com/HyperPlay-Gaming/product-management/issues/188#issuecomment-1981607149

**Comment by iamdavid0091** [comment]
@red-game-dev here is the [Figma link](https://www.figma.com/file/GKPELZxk551oVC3EDLgEDA/Current-Design?type=design&node-id=9848%3A2021&mode=design&t=fHJ81AIIx72cgk6P-1) you requested.

**Comment by BrettCleary** [comment]
@red-game-dev 

1. The "Chrome" text should be neutral-400
2. The top and bottom padding on the browser import button should be larger
3. There is too much spacing between the metamask icons and the text below them for "Import Using Secret Recovery Phrase" and "Create New Extension Wallet"
4. The browser icon is 24x24 px on the design but 32x32 px on this branch

image

**Comment by red-game-dev** [comment]
> @red-game-dev comparing this to https://www.figma.com/file/GKPELZxk551oVC3EDLgEDA/Current-Design?type=design&node-id=9848-73492&mode=design&t=KeWWWRtJVeKKwsxy-4
> 
> 1. The "Chrome" text should be neutral-400
> 2. The top and bottom padding on the browser import button should be larger
> 3. There is too much spacing between the metamask icons and the text below them for "Import Using Secret Recovery Phrase" and "Create New Extension Wallet"
> 4. The browser icon is 24x24 px on the design but 32x32 px on this branch
> 
> image

Sorted, cheers for the helpful list of issues 🙏 🙌

**Review by BrettCleary — APPROVED** [review]
LGTM

**Review by eliobricenov — APPROVED** [review]

**Pull request merged**